File: src/main/scala/org/openapitools/server/model/GetChatsInfoDTO.scala
package org.openapitools.server.model


/**
 * Список чатов.
 *
 * @param chats Информация о чатах. for example: ''null''
 * @param paging  for example: ''null''
*/
final case class GetChatsInfoDTO (
  chats: Seq[GetChatInfoDTO],
  paging: Option[ForwardScrollingPagerDTO] = None
)

File: ympa_fsharp_giraffe_server/src/model/GetChatsInfoDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.ForwardScrollingPagerDTO
open ympa_fsharp_giraffe_server.Model.GetChatInfoDTO

module GetChatsInfoDTO =

  //#region GetChatsInfoDTO


  type GetChatsInfoDTO = {
    Chats : GetChatInfoDTO[];
    Paging : ForwardScrollingPagerDTO;
  }
  //#endregion
